<launch>
    <arg name="gdb" default="false" doc="start the world model in gdb" />
    <group if="$(arg gdb)">
        <node pkg="bitbots_world_model" type="local_ball_filter" name="local_ball_filter" output="screen" launch-prefix="gdb --args">
            <rosparam command="load" file="$(find bitbots_world_model)/config/local_ball_filter_config.yaml"/>
        </node>
    </group>
    <group unless="$(arg gdb)">
        <node pkg="bitbots_world_model" type="local_ball_filter" name="local_ball_filter" output="screen">
            <rosparam command="load" file="$(find bitbots_world_model)/config/local_ball_filter_config.yaml"/>
        </node>
    </group>
</launch>
